modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -488,8 +488,8 @@ GrainTracker::remapGrains()
       if (grain_it1->second._status == Status::INACTIVE)
         continue;
 
-      // If we have the reserve op flag set, see if we need to move any grains immediately
-      if (_reserve_op && grain_it1->second._var_idx == 0) // The zeroth variable is always the reserved variable
+      // We need to remap any grains represented on any variable index above the cuttoff
+      if (grain_it1->second._var_idx >= _reserve_cutoff)
```

Comment:
Yeah - I fixed this already - just haven't pushed up a new version since you need more than this. Same with the comment above.